XLPandXeLaTeX.ltx

```diff
@@ -46,74 +46,69 @@ producing complex documents in beautiful formats. \TeX
 pages (HTML 4), Microsoft Word (.doc), Open Office Writer (.odt), and ePUB. It automatically
-numbers sections, figures, tables, and examples. It keeps track of internal references to these entites along with citation references and gloss abbreviations.  This keeps numbering and reference links dependable and automated. It also
+numbers sections, figures, tables, and examples. It keeps track of internal references to these entites along with citation references, and glosses/abbreviations keeping numbering and reference links dependable and automated. It also
 automatically generates abbreviations used and references cited (using a custom references implimentation).
 
-Unlike most editing programs which are based on either the WYSIWYG paradigm or as text editors used to code or produce markdown, XLingPaper (via the XMLmind XML Editor)
-is a structured editor. Rather than making the document look the way
+Unlike most editing programs which are based on either the \WYSIWYG paradigm or as text editors used to code or produce markdown, XLingPaper (via the \XMLmind \XML Editor)
+is a structured editor. Rather than visually structuring the document look the way
 it is to be formatted, the author “marks up” the items in the document
-according to their kind.  One of the many values this gives is that
+according to their styling needs.  One of the many values this gives is that
 there is a “grammar” of what a well-formed linguistic document looks
-like.  This makes it easy to move and shift sections and chapters and
-examples and prevents users from inadvertently creating ill-formed
+like.  This makes moving, replacing, switching, or reordering sections, chapters, and
+examples less error prone becasue it prevents users from inadvertently creating ill-formed
 documents.
 
-This paper is about the \XeLaTeX{}-produced \PDF{} output.  
-
-The official website for XLingPaper is
-\tbsurl{https://software.sil.org/xlingpaper}.  A list of advantages for the
-author, the reader, and the publisher can be found at the “Why
-XLingPaper?”\cite{black:2017} portion of the documentation.
+The following sections of this paper discuss the\TeX{} technologies used. 
 
 \section{XLingPaper and \TeX{}}
 
-We now describe how XLingPaper uses \TeX{} technologies.
+Due to the heavy reliance on Unicode in modern language documentation and linguistic work, XLingPaper specifically uses \XeLaTeX{} and compatible packages to produce \PDF{} outputs.  The following sections provide more detial on the design requirments and packages used.
 
 \subsection{Design desiderata for \TeX{} with XLingPaper}
 
-From the outset, XLingPaper was designed to be free.  The XMLmind XML Editor had a Personal
-Use License that fit the bill for the vast majority of the target
+From the outset, XLingPaper was designed to be free.  The XMLmind XML Editor had a Personal Use License that fit the bill for the vast majority of the target
```

Comment:
The difference here is line length.  The requirements for submission had something about limiting line length to 80 characters or some such.  I'm using Emacs which has an easy way to do that.

Reply:
I am using gedit on linux. If I have that feature I am not sure how to use it. Maybe the solution is for me to just not worry about line length (long or short) until the end and then send it to you for final processing?

Reply:
Sure.  I'd be happy to do that.